### DISCUSSION TITLE
Polygon Intersection Vertices

### DISCUSSION THREAD

**Discussion by AssisiSoft** [NetTopologySuite/NetTopologySuite#673] [Q&A]
When intersecting polygons, why does the resulting polygon have only the lowest number of source vertices rather than all the vertices of the intersection?  For example, intersecting a 6 point polygon with a 100 point polygon returns a 6 point polygon.  How does one get the intersected polygon with all the vertices from both polygons that fall within the intersection?

Am I confusing "dimensions" with "vertices"?

**Comment by FObermaier** [comment]
I don't understand.
* P0:
  `POLYGON ((30 10, 10 20, 20 40, 40 40, 30 10))`
* P1:
  `POLYGON ((21 -1, 21 7, 21 18, 23 19, 24 22, 26 21, 29 24, 30 21, 33 24, 36 23, 37 16, 40 7, 30 -2, 21 -1))`
* P0 &#8745; P1:
  `POLYGON ((30 10, 21 14.5, 21 18, 23 19, 24 22, 26 21, 29 24, 30 21, 33 24, 34.5 23.5, 30 10))`

**Reply by AssisiSoft** [reply]
Setting PrecisionModel.Scale=1,000,000 returned full set of vertices.

Does PrecisionModel.Scale determine the resolution of intersecting testing?  IE, something like this:  Scale = 100 test vertices with X or Y differences of 2 significant digits and Scale = 1,000,000 tests vertices with 6 significant digits?

**Reply by FObermaier** [reply]
The precision model is used for all computations of `Coordinate`s and when serializing the `Geometry`s.
